### DISCUSSION TITLE
Is it possible to add text input fields to a notification?

### DISCUSSION THREAD

**Discussion by danimesq** [phuhl/linux_notification_center#210] [Q&A]
For example, replying to a message right from the notification?

**Comment by phuhl** [comment]
Hi @danimesq,

the [Desktop Notifications Specification](https://specifications.freedesktop.org/notification-spec/notification-spec-latest.html) does not specify any way to do something like this. That means, there will most likely not be any applications that would support this behavior as it would be based on some kind of a custom protocol.

Are you aware of any applications where this works?

**Reply by ahmubashshir** [reply]
Telegram in KDE, using native notifications.
